core/scripts/functions/templates/oracle.toml

```diff
@@ -67,4 +67,8 @@ maxSecretsSizesList = [10_240, 20_480, 51_200, 102_400, 307_200, 512_000, 1_048_
   NodeAddress = "{{node_eth_address}}"
 
     [pluginConfig.gatewayConnectorConfig.WsClientConfig]
-    HandshakeTimeoutMillis = 1_000
\ No newline at end of file
+    HandshakeTimeoutMillis = 1_000
+
+    [[pluginConfig.gatewayConnectorConfig.Gateways]]
+    Id = "gateway"
+    URL = "wss://"
```

Comment:
Maybe makes most sense to do:
  Id = "{{gateway_id}}"
  URL= "{{gateway_url}}"
  
And modify the script to have those two extra vars?